spock-core/src/main/java/org/spockframework/compiler/WhereBlockRewriter.java

```diff
@@ -16,19 +16,22 @@
 
 package org.spockframework.compiler;
 
-import java.util.*;
-
 import org.codehaus.groovy.ast.*;
 import org.codehaus.groovy.ast.expr.*;
-import org.codehaus.groovy.ast.stmt.*;
```

Comment:
Left it at the default Idea setting, as there was no formatter committed to the project.
Will revert unrelated imports, if it bothers the reviewers.